dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletCacheImpl.java

```diff
@@ -49,122 +49,122 @@ public TranslatedQuery getTranslatedQuery(String key) {
 		if(o==null){
 			return null;
 		}else{
-			return (TranslatedQuery)o;	
-		} 
+			return (TranslatedQuery)o;
+		}
 	}
-	
+
 	@Override
 	public void addMetadata(String key, String metadata) {
-	    key = metadataGroup + key;
-	    if(!UtilMethods.isSet(metadata))
-	        metadata=EMPTY_METADATA;
-	    cache.put(key, metadata, metadataGroup);
+		key = metadataGroup + key;
+		if(!UtilMethods.isSet(metadata))
+			metadata=EMPTY_METADATA;
+		cache.put(key, metadata, metadataGroup);
 	}
-	
+
 	@Override
 	public void addMetadata(String key, com.dotmarketing.portlets.contentlet.model.Contentlet content) {
-	    // http://jira.dotmarketing.net/browse/DOTCMS-7335
-        // we need metadata in other cache region
+		// http://jira.dotmarketing.net/browse/DOTCMS-7335
+		// we need metadata in other cache region
 		if("CACHE_404_CONTENTLET".equals(content.getInode())){
 			return;
 		}
-        Structure st=content.getStructure();
-        if(st!=null && st.getStructureType()==Structure.STRUCTURE_TYPE_FILEASSET) {
-            Field f=st.getFieldVar(FileAssetAPI.META_DATA_FIELD);
-            if(f!=null && UtilMethods.isSet(f.getInode())) {
-                String metadata=(String)content.get(FileAssetAPI.META_DATA_FIELD);
-                addMetadata(key, metadata);
-                content.setStringProperty(FileAssetAPI.META_DATA_FIELD, ContentletCache.CACHED_METADATA);
-            }
-        }
+		Structure st=content.getStructure();
+		if(st!=null && st.getStructureType()==Structure.STRUCTURE_TYPE_FILEASSET) {
+			Field f=st.getFieldVar(FileAssetAPI.META_DATA_FIELD);
+			if(f!=null && UtilMethods.isSet(f.getInode())) {
+				String metadata=(String)content.get(FileAssetAPI.META_DATA_FIELD);
+				addMetadata(key, metadata);
+				content.setStringProperty(FileAssetAPI.META_DATA_FIELD, ContentletCache.CACHED_METADATA);
+			}
+		}
 	}
-	
+
 	@Override
 	public String getMetadata(String key) {
-	    key = metadataGroup + key;
-	    String metadata=null;
-	    try {
-            metadata=(String)cache.get(key, metadataGroup);
-        } catch (DotCacheException e) {
-            Logger.debug(this, "Cache Entry not found", e);
-        }
-	    return metadata;
+		key = metadataGroup + key;
+		String metadata=null;
+		try {
+			metadata=(String)cache.get(key, metadataGroup);
+		} catch (DotCacheException e) {
+			Logger.debug(this, "Cache Entry not found", e);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Cache Entry not found" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)